pkg/networkservice/chains/nsmgr/suite_test.go

```diff
@@ -394,42 +397,56 @@ func (s *nsmgrSuite) Test_PassThroughRemoteUsecase() {
 	ctx, cancel := context.WithTimeout(context.Background(), time.Second*5)
 	defer cancel()
 
+	labels := []string{"a", "b", "c", "d", "e", "f", "g"}
```

Comment:
I suppose that `nodesCount` or `nsesCount` is enough for the linear case, there is no need for the `labels` list.
You can simply create `linearNS(count)` with `step: i --> step: i+1` routes.
And so it is the same for the NSE registration loop, there is no need for `labels[i]` when you can simply use `i`.